index.html

```diff
@@ -81,7 +81,7 @@ <h1 class="project__heading poppins">
                 </ul>
 
                 <div>
-                    <button type='button' class="project__button poppins">
+                    <button type='button' class="project__button poppins" aria-label="See Project">
                         see Project
                     </button>
```

Comment:
![color2](https://user-images.githubusercontent.com/93039359/187473898-077510f0-ec0e-4614-b30b-d2725391c06a.png)

- [ ] Your see project button should have a color contrast of 4.5:1 ratio due to accessibility purpose Please make this change :+1: